repository/src/main/java/org/apache/atlas/tasks/AtlasTaskService.java

```diff
@@ -162,6 +162,8 @@ public List<AtlasTask> createAtlasTasks(List<AtlasTask> tasks) throws AtlasBaseE
                 task.setCreatedTime(new Date());
                 task.setStatusPending();
                 task.setAttemptCount(0);
+                task.setAssetsCountToPropagate(task.getAssetsCountToPropagate());
```

Comment:
At this point, won't the `assetsCountToPropagate` be 0? Do we need to call `getAssetsCountToPropagate()`?

Reply:
Going point. reverted to `0L`